### PR TITLE
[IMP] l10n_pe: Welcome PLE reports 

### PR DESCRIPTION
**Pull request by luistorresm**
Complement module to allow generating the PLE reports.

- Missing purchase tax groups and taxes added to complete the vendor
  bills flow
- Demo data completed to has data to allow generate the PLE reports
- Document types for vendor bills completed in the data
- Method to improve the domain in document types for customer invoices
  added
- Extend the method to autocomplete the number on vendor bills with the
  standard expected by the document type.

### CODE REVIEW

File: addons/l10n_latam_invoice_document/models/account_move.py [hunk 1]
```diff
@@ -214,15 +214,19 @@ def _check_unique_vendor_number(self):
...
-                ('move_type', '=', rec.move_type),
-                # by validating name we validate l10n_latam_document_type_id
-                ('name', '=', rec.name),
-                ('company_id', '=', rec.company_id.id),
-                ('id', '!=', rec.id),
-                ('commercial_partner_id', '=', rec.commercial_partner_id.id),
-                # allow to have to equal if they are cancelled
-                ('state', '!=', 'cancel'),
-            ]
+            domain = rec._get_domain_unique_vendor_number()
```
Comment (jbw-odoo) [inline addons/l10n_latam_invoice_document/models/account_move.py:217]: Why do you create this method ?

**Review by jbw-odoo — COMMENTED** [review]
Hi !
Thank you for contributing.
@jco-odoo can you have a look ?

File: addons/l10n_latam_invoice_document/models/account_move.py (hunk 1 above)
Comment (jco-odoo) [reply addons/l10n_latam_invoice_document/models/account_move.py:217]: @luistorresm It is probably not necessary to inherit this anymore.  It was taken into account for the _sequence module afaik.

**Comment by jco-odoo** [comment]
@jbw-odoo This still WIP anyways.

**Comment by jbw-odoo** [comment]
> @jbw-odoo This still WIP anyways.
Ah ok sorry :)

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**